microsoft/testsuites/ltp/ltp.py

```diff
@@ -125,7 +125,8 @@ def run_test(
         self.run(parameters, sudo=True, force_run=True, timeout=12000)
 
         # to avoid no permission issue when copying back files
-        self.node.tools[Chmod].update_folder("/opt", "a+rwX", sudo=True)
+        self.node.tools[Chmod].update_folder("/opt/ltp", "a+rwX", sudo=True)
```

Comment:
Do we need this now?

Reply:
I will try without it and remove it if no longer neccessary.